chrome/style.css

```diff
@@ -68,15 +68,205 @@
     outline: none; 
 }
 
+#gttButton > div {
+    opacity:0.55;
+}
+
+#gttButton:hover > div {
+    opacity:1;
+}
+
 #gttButton * {outline:none}
-#gttButton img {vertical-align: -4px; height: 16px; padding-right:2px;}
+
+#gttButton img {
+    vertical-align: -3px;
+    height: 14px;
+    width: 14px;
+    padding-right: 2px;
+}
+
+#gttButton #gttDownArrow {
+    opacity: 1;
+    background: none;
+    font-size: 12px;
+    line-height: 5px;
+}
+
+#gttButton.withCards {
+    background-color: #0f9d58;
+    background-image: linear-gradient(to bottom,#0f9d58,#0f9152);
+    color: #ffffff;
+}
+
+#gttButton.withCards:hover {
+    background-image: linear-gradient(to bottom,#0f9d58,#058545);
+    border: 1px solid #007038;
+}
+
+#gttButton.withCards > div {
+    opacity:1;
+}
+
+#gttButton.withCards:hover > div {
+    opacity:1;
+}
+
+#gttButton.withCards img {
+    filter: invert(100%);
+}
```

Comment:
Totally restyled the gttButton. It's grey with a grey icon when you have no attached Trello Cards. When there are cards attached, it turns green with a white icon. I tried to use the standard Google colors for the green button, although I don't think Google has a green button like this so it's made up.